docs/examples/plot_backtracking_sloped_terrain.py

```diff
@@ -0,0 +1,176 @@
+"""
+Backtracking on sloped terrain
+==============================
+
+Modeling backtracking for single-axis tracker arrays on sloped terrain.
```

Comment:
This reads like a module docstring but renders as regular text, so I think the rendered introduction is awkward.

Reply:
Agreed it's awkward in the rendered page.  The reason for the strange wording is the docstring-style line is also used as the hover text for the thumbnail in the gallery listing, which has to be brief.  Open to suggestions for text that works better in both contexts!

Reply:
Sounds like a gallery configuration problem. Let's ignore.